src/index.css

```diff
@@ -24,7 +28,13 @@ a:hover {
 }
 
 body {
+  background-image: url('/bg.png');
```

Comment:
The problem is that this CSS rule is applied to the body element, affecting the background of all pages which is not required

Reply:
design is  just needed in landing page ??

Reply:
@Gautam-Hegde  what else is  needed??

Reply:
> design is just needed in landing page ??

yep